app/model/Config.scala

```diff
@@ -10,6 +10,11 @@ case class Config(conf: Configuration) {
   val mavenLocalPlugins = Site(conf.get[Configuration]("maven.local-plugins"))
   val mavenCentral = Site(conf.get[Configuration]("maven.central"))
   val mavenGradlePlugins = Site(conf.get[Configuration]("maven.gradle-plugins"))
+  val localRepositories: Seq[String] = getLocalRepositories
+
+  private def getLocalRepositories: Seq[String] = {
+    conf.get[Configuration]("local").get("repositories")(ConfigLoader.seqStringLoader)
```

Comment:
Would it be better to use `getOptional` instead ? Would be nicer if we don't have to explicitly add it in the configuration even when empty 😛

Reply:
done